pdctl/command/store_command.go

```diff
@@ -47,6 +49,16 @@ func NewDeleteStoreCommand() *cobra.Command {
 	return d
 }
 
+// NewLabelStoreCommand returns a label subcommand of storeCmd.
+func NewLabelStoreCommand() *cobra.Command {
+	l := &cobra.Command{
+		Use:   "label <store_id> <key> <value>",
+		Short: "set a store's label value",
```

Comment:
Seems this command actually add labels to the store.
Can we add sub commands to get/set/add labels to a store?